locale/en/blog/release/v10.0.0.md

```diff
@@ -113,7 +113,7 @@ The following APIs have been deprecated in Node.js 10.0.0
 * Previously deprecated internal getters/setters on `net.Server` has reached end-of-life and have been removed. [[`3701b02309`](https://github.com/nodejs/node/commit/3701b02309)]
 * Use of non-string values for `process.env` has been deprecated in documentation. [[`5826fe4e79`](https://github.com/nodejs/node/commit/5826fe4e79)]
 * Use of `process.assert()` will emit a runtime deprecation warning. [[`703e37cf3f`](https://github.com/nodejs/node/commit/703e37cf3f)]
-* Previously deprecated `NODE\_REPL\_HISTORY\_FILE` environment variable has reached end-of-life and has been removed. [[`60c9ad7979`](https://github.com/nodejs/node/commit/60c9ad7979)]
```

Comment:
I think back-slash is required here for website.
If you check here: https://nodejs.org/en/blog/release/v10.0.0/
back-slash doesn't appear.

Reply:
They are not needed if they are in the code backticks and in this case they appear (try to search with them):

![bb](https://user-images.githubusercontent.com/10393198/39217986-01d988b2-482b-11e8-92be-f342072189e0.png)

Reply:
You are right. Sorry.
Updated the review. 👍